packages/react-card/package.json

```diff
@@ -25,7 +25,7 @@
     "storybook": "start-storybook"
   },
   "devDependencies": {
-    "@fluentui/babel-make-styles": "^9.0.0-alpha.34",
+    "@fluentui/babel-make-styles": "^9.0.0-alpha.35",
```

Comment:
shouldn't this be automated ?

Reply:
race condition in CI means that PRs can sometimes pass with mismatching versions unless you run the pipeline again

Reply:
I see, until we wont enable always up to date PR this will be an issue indefinitely. thx for quick fix !